app/tests/TestCase.php

```diff
@@ -61,7 +60,7 @@ protected function sort(array &$array): void
 
     protected function isValidation(string $filename): bool
     {
-        return Str::startsWith($filename, 'validation');
+        return str_contains($filename, 'validation');
```

Comment:
str_starts_with ?

Reply:
Yeah sorry. My mistake. I need to sleep.

Reply:
The code checks the occurrence of words on the file name, so you can use both the `str_starts_with` function and `str_contains`. Taking into account our implementation, changing them is tantamount to saving on matches. We can use both, but I prefer to write correctly right away. The file names `validation.php` and `validation-inline.php` always start with the `validation` word, so it is better to use the `str_starts_with` function.